src/tree/layout.rs

```diff
@@ -4,6 +4,9 @@ use crate::style::AvailableSpace;
 use crate::style_helpers::TaffyMaxContent;
 use crate::util::sys::{f32_max, f32_min};
 
+#[cfg(feature = "detailed_layout_info")]
```

Comment:
This needs to be feature flagged on grid feature. Alternatively, just use full path directly below.

Reply:
Alright, full path struct would be very nice.